### ISSUE TITLE
CUDA 10.1 is not compatible with newer GPUs

### ISSUE THREAD

**Issue by MineThingIssues** [harskish/ganspace#55]
I have used ganspace with stylegan2 back when I was on an older system using a GTX 1060 6GB, however I have recently upgraded to an RTX 3060 Ti and have run into issues with CUDA 10.1 compatibility.

I am using Windows 10

I have tried to fix this by installing CUDA Toolkit 11.1.1 and changing environment.yml to 11.1 then restarting the installation process, however this lead to some pytorch issues, and I believe that pytorch 1.7.1 or newer is required for cuda 11.1 compatibility.
I also tried installing pytorch 1.7.1 and 1.10.0, but this lead to even more issues and so I thought I'd come here for some advice on how to proceed.

I am not very experienced when it comes to this area so apologies if I get stuck at any point or don't understand something

**Comment by 2blackbar** [comment]
yeah, usual python styuff ! deprecated afgter couple of months, i so wish they would stop using the damn python for machine learning ! This is insane, a lot of envs are broken after couple of months, backwards comp is a joke over there, python is always a language for bored teen stuydents that didnt want to learn C
